src/index.test.ts

```diff
@@ -74,8 +78,8 @@ describe('public sarifReporter API', () => {
     // Normalized values are the pinned expectations from basic-axe-v3.2.2-sarif-v2.1.2.sarif
     function normalizeEnvironmentDerivedSarifProperties(sarif: SarifLog): void {
         sarif.runs[0]!.invocations!.forEach(i => {
-            i.endTimeUtc = '2019-03-22T19:12:06.129Z';
-            i.startTimeUtc = '2019-03-22T19:12:06.129Z';
+            i.endTimeUtc = '2000-01-02T03:04:05.006Z';
```

Comment:
this utc time string seems to show up on a lot of places... except for snapshots, does it make sense to move it to a `const` that can be imported from other places?

Reply:
Most of the places are in test-resources files; the only "code" places are here and at one spot in the generator. I think it's reasonable for those 2 spots to be pinned to each other, though (the generator is already importing code from the main library anyway); will make that update.